src/components/SearchGridManualLoad.vue

```diff
@@ -208,8 +208,7 @@ export default {
   .search-grid-cells {
     display: flex;
     flex-wrap: wrap;
-    margin: 10px;
-
+    margin: -10px;
```

Comment:
This causes the grid to look broken on the results page. I think you need a different solution for this

<img width="1775" alt="Screen Shot 2020-05-25 at 11 15 28" src="https://user-images.githubusercontent.com/707019/82820704-0c21e200-9e79-11ea-9ba0-4cd1f5faca0b.png">

Reply:
@brenoferreira Thank you for reviewing my pr. I have made one more change please look into it, if this could resolve the issue.